test/integration/test-one.sh

```diff
@@ -36,6 +36,7 @@ RUNDIR=$(_CS_DARWIN_USER_TEMP_DIR='' TMPDIR='' mktemp -d /tmp/spire-integration-
 export ROOTDIR
 export REPODIR
 export RUNDIR
+export TESTNAME
```

Comment:
is this used?

Reply:
I use it to determine the name of the test to see if the events based cache should be enabled or not. The `events-based-entries` suite is just a symlink to the `entries` suite. The `entries` suite determines based on the name if the events are enabled or not.